### PR TITLE
Archive page

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes 

- Archive page 
- User can archive and unarchive page

### CODE REVIEW

File: src/components/NoteCard/NoteCard.tsx [hunk 1]
```diff
@@ -84,7 +84,10 @@ export const NoteCard = ({
               )}
           </div>
           <button className="mr-5">
-            <MdOutlineArchive title="Archive" />
+            <MdOutlineArchive
+              title="Archive"
```
Comment (HarshnaNHaswani) [inline src/components/NoteCard/NoteCard.tsx:88]: what's md?

**Review by HarshnaNHaswani — APPROVED** [review]
looks good to do
- neat folder structure
- clear code
- easy to understand data flow

File: src/components/NoteCard/NoteCard.tsx (hunk 1 above)
Comment (junaidshaikh-js) [reply src/components/NoteCard/NoteCard.tsx:88]: Provided by react-icons

**Review by rahulyadav139 — APPROVED** [review]
Everything is looking great. I didn't find anything to correct. Great work.

**Pull request merged**